wagtail_transfer/operations.py

```diff
@@ -525,31 +531,31 @@ def dependencies(self):
             if isinstance(field, models.ForeignKey):
                 val = self.object_data['fields'].get(field.name)
                 if val is not None:
-                    # TODO: consult config to decide whether objective type should be 'exists' or 'updated'
+                    objective_type = 'updated' if field.related_model._meta.label_lower in UPDATE_RELATED_MODELS else 'exists'
```

Comment:
Would it be worth lowering the labels extracted from the setting as well, if you're checking against label_lower rather than label? (eg when you populate UPDATE_RELATED_MODELS)

Reply:
Good plan! I initially wanted to convert UPDATE_RELATED_MODELS into a list of classes up-front, so that we could write `if field.related_model in UPDATE_RELATED_MODELS` here - but then you run into problems with the app startup sequence (you can't look up the names until the contenttypes table has been populated). Normalising the names to lowercase is a good middle ground.